src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefCreateTest.java

```diff
@@ -61,6 +62,11 @@ public void whenUserNotExistsInCwrAndSidamAndUp_Ac1() {
         response.then()
                 .assertThat()
                 .statusCode(201);
+
+        CaseWorkerProfileCreationResponse caseWorkerProfileCreationResponse =
+                response.getBody().as(CaseWorkerProfileCreationResponse.class);
+        caseWorkerIds = new ArrayList<>(caseWorkerProfileCreationResponse.getCaseWorkerIds());
```

Comment:
do we need new array List?

Reply:
I shall remove and do something like `caseWorkerIds = caseWorkerProfileCreationResponse.getCaseWorkerIds();`